src/main/resources/plugin.yml

```diff
@@ -14,4 +14,4 @@ permissions:
     fm.public:
         default: true
     fm.op:
-        default: op
\ No newline at end of file
+        default: op
```

Comment:
These perms are FactoryMod's (because I copied the plugin.yml) from there and we should probably change them. Doesnt have to be in this PR, but should keep it in mind.

Reply:
Hmm that's probably related to the issue I had earlier with the commands. `/help MemeMana` was showing a few FactoryMod commands 🤔 . Do you need to list the commands your plugin provides in `plugin.yml`?

Reply:
Yeah, you need to register the commands in the plugin.yml and ideally also assign them a permission.